src/main/java/ca/intelliware/ihtsdo/mlds/repository/PageableUtil.java

```diff
@@ -1,10 +1,15 @@
 package ca.intelliware.ihtsdo.mlds.repository;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.data.domain.Pageable;
 
 public class PageableUtil {
+	static final Logger LOG = LoggerFactory.getLogger(PageableUtil.class);
+
 	public int getStartPosition(Pageable pageable) {
-		return (pageable.getPageNumber() * pageable.getPageSize()) + pageable.getOffset();
+		//return (pageable.getPageNumber() * pageable.getPageSize()) + pageable.getOffset(); // MLDS-967, start position increases too fast
+		LOG.debug("start position={}", pageable.getPageNumber() * pageable.getPageSize());
```

Comment:
Did you want to leave that in here?   Is our logging level in Prod at "Info" ?   Fine to leave debug in if so, but I would give it as much context as you can ... that may not be easy in a utility class.    The main thing is to imagine someone other than you seeing this in the logs, what would they make of it?

Reply:
Thanks, I changed the text - I think it's only useful for debugging.